src/main/resources/spy.properties

```diff
@@ -0,0 +1,5 @@
+# We use P6Spy for logging JDBC operations in Sentry. We exclude everything else from the module list
+# (mainly the P6LogFactory) so that we don't flood our system with P6Spy log files that we don't use.
+# https://blog.sentry.io/troubleshooting-spring-boot-applications-with-sentry/#jdbc-instrumentation
+modulelist=com.p6spy.engine.spy.P6SpyFactory
+driverlist=com.mysql.jdbc.Driver
```

Comment:
Do we still need this if p6spy is configured on the server?

Reply:
Enabling p6spy via:
```
dataSource:
    driverClassName: com.p6spy.engine.spy.P6SpyDriver
    url: jdbc:p6spy:mysql://localhost:3306/openboxes?serverTimezone=UTC&useSSL=false
```
still requires the settings here for p6spy to be able to know what "real" driver to forward to after it does its thing.

from what I saw when implementing this, everyone was configuring this via spy.properties. I can't remember if it was a requirement for it to work or if it was smart enough to look in the application.yml file. I can test it out again.

Reply:
Yeah that's what i was thinking. Not a big deal. I just didn't want to keep a file around that wasn't necessary.